docs/docs-content/release-notes/known-issues.md

```diff
@@ -16,6 +16,7 @@ The following table lists all known issues that are currently active and affecti
 | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ | ---------------------------- |
+| Self-hosted instances of Palette in VMware vCenter do not have unique cluster IDs for their persistetn volumes. This can lead to issues if there are multiple self-hosted instances of Palette or during an upgrade.                                                                                                                                                                                                                                                                                                                                                     | Refer to the Troubleshooting section for guidance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Sep 13, 2024       | Self-hosted                  |
```

Comment:
Double checking on Sumit with the issue because I don't think clusters not having unique IDs was the case, but rather the PVC all looked the same, regardless of what EC cluster it belonged to.

Reply:
I am just going to use the language from Romain in the slack channel